pkg/builder/builder.go

```diff
@@ -286,8 +293,12 @@ func buildVolumes(vdb *vapi.VerticaDB) []corev1.Volume {
 	if vdb.GetSSHSecretName() != "" {
 		vols = append(vols, buildSSHVolume(vdb))
 	}
-	if vdb.Spec.NmaTLSSecret != "" {
-		vols = append(vols, buildHTTPServerSecretVolume(vdb))
+	if vmeta.UseVClusterOps(vdb.Annotations) {
+		if vmeta.UseNMACertsMount(vdb.Annotations) {
```

Comment:
Could we combine the lines?
```suggestion
	if vmeta.UseVClusterOps(vdb.Annotations) && vmeta.UseNMACertsMount(vdb.Annotations) {
```